website/src/content/docs/docs/libraries/typespec-client-generator-core/reference/decorators.md

```diff
@@ -1177,3 +1177,43 @@ model SportsCar extends Vehicle {
 }
 
 ```
+
+### `@markAsLro` {#@Azure.ClientGenerator.Core.Legacy.markAsLro}
+
+Forces an operation to be treated as a Long Running Operation (LRO) by the SDK generators,
+even when the operation is not long-running on the service side.
+
+When applied, TCGC will treat the operation as an LRO and SDK generators should:
+
+- Generate polling mechanisms (pollers)
+- Return appropriate LRO-specific return types
+- Handle the operation as an asynchronous long-running process
+
+This decorator is considered legacy functionality and should only be used when
+standard TypeSpec LRO patterns are not feasible.
```

Comment:
I would add a bigger warning blurb about the requirement to verify the operation/add tests for the generated code. Else, there is a definite risk that it gets added to operations that only worked by accident in autorest classic.